tests/modules/tailfindr/main.nf

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { TAILFINDR } from '../../../modules/tailfindr/main.nf'
+
+workflow test_tailfindr {
+    
+    input = [
+        [ id: 'test' ], // meta map
+        file(params.test_data, checkIfExists: true)
```

Comment:
`params.test_data` needs to be changed by referring to a file in the test-dataset, e.g. https://github.com/nf-core/modules/blob/7e8ad566883449e7939062b5e2bcf53fc1e0002f/tests/modules/artic/minion/main.nf#L12